addons/http_routing/views/http_routing_template.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="http_error">
+        <t t-call="web.frontend_layout">
+            <div id="wrap">
+                <div class="oe_structure">
+                    <h1 class="container mt32"><t t-esc="status_code"/>: <t t-esc="status_message"/></h1>
+                </div>
+                <t t-if="editable or debug">
```

Comment:
somehow `debug` seems to be always true (was already the case before your commit.. checked in 12.4 and same bug) while `request.session.debug` has the correct value.. (it should be the same as debug is an alias to request.session.debug added in prepare_qcontext)
So the traceback is always shown and not only in debug
Maybe we can change it here so 12.5 is clean and I'll investigate in 12.4 what's going on